src/index.html

```diff
@@ -22,6 +26,412 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="container_logo"
+      >
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+      <nav class="container_menu">
+        <ul class="menu">
+          <li>
+            <a
+              class="is-active"
+              href="#"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
```

Comment:
The 'data-qa' attribute is used for testing purposes and should not be present in the production code unless it's being specifically used for test automation.